Kconfig

```diff
@@ -20,9 +20,19 @@ config HP_MEMORY_BANKS
 config LP_MEMORY_BANKS
 	int "LP memory banks count"
 	default 1
+	depends on CAVS
 	help
 	  Available memory banks count for Low Power memory
 
+config LP_SRAM
+	bool "Enable LPSRAM memory"
+	default n
+	depends on CAVS
+	help
+	  Turn ON/OFF LPSRAM bank/s. This provides an option to enable an LPSRAM
+	  memory for specific platform. Until this options is enabled firmware will
+	  not enable any LPSRAM memory bank and all related defines will return zero.
```

Comment:
I think you may remove ` and all related defines will return zero`, it's better not to write implemenation details, especially when this statement is not true, because it will result in different (0/false/undefined) values depending in what enviroment config is used (fe cmake/make vs C preprocessor vs C compiler).